### ISSUE TITLE
TST: pyinstaller job failed with No module named 'mypackage'

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#12999]
I think setuptools 61.0.0 that got released yesterday might have broken this? @nstarman , is it time to move this out of core?

https://github.com/astropy/astropy/runs/5686402514?check_suite_focus=true

```
________________________ ERROR collecting test session _________________________
_pytest/config/__init__.py:607: in _importconftest
    ???
_pytest/pathlib.py:533: in import_path
    ???
importlib/__init__.py:127: in import_module
    ???
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
_pytest/assertion/rewrite.py:168: in exec_module
    ???
astropy_tests/cosmology/flrw/tests/conftest.py:5: in <module>
    from astropy.cosmology.tests.helper import clean_registry
PyInstaller/loader/pyimod03_importers.py:495: in exec_module
    ???
astropy/cosmology/tests/__init__.py:7: in <module>
    ???
E   ModuleNotFoundError: No module named 'mypackage'
```

**Comment by nstarman** [comment]
Yes. I think we settled on ``astropy-data`` as a temporary home?

**Comment by pllim** [comment]
I thought it was going to go to https://github.com/astropy/integration-testing ? 🤷 

@bsipocz or @astrofrog , do you remember?

**Comment by pllim** [comment]
Oh, wait... was it going to be its own dummy package somewhere? I thought you set something up with @hamogu ?

**Comment by pllim** [comment]
@nstarman , unless you know a simple fix for this to get pyinstaller green again, can we just remove it now and figure out where to move it to separately afterwards? 😬

**Comment by nstarman** [comment]
I guess as an astropy infrastructure package. Make it a test dependency or something.

**Comment by pllim** [comment]
I am thinking the other way around, the fake package would have CI set up to test against stable and dev astropy.

**Comment by nstarman** [comment]
Hm.. that could be good. But then things can be merged into astropy main that break older astropy. I suppose then I'd have to lodge a bug report, etc.

**Comment by pllim** [comment]
Until https://github.com/astropy/integration-testing takes off, yes...

**Comment by pllim** [comment]
@nstarman , can I go ahead and remove it here or you want to do it but waiting on something?

**Comment by nstarman** [comment]
Go for it, thanks!

**Comment by pllim** [comment]
See #13018
